tap_exacttarget/endpoints/lists.py

```diff
@@ -55,7 +55,7 @@ def sync_data(self):
         selector = FuelSDK.ET_List
 
         search_filter = None
-        retrieve_all_since = get_last_record_value_for_table(self.state, table)
+        retrieve_all_since = get_last_record_value_for_table(self.state, table, self.config)
```

Comment:
Please comment as to what the changes are? I don't see any comments being added.

Reply:
Added comment